security-analysis/src/main/java/eu/itesla_project/security/LimitViolation.java

```diff
@@ -32,7 +35,10 @@
 
     private final float baseVoltage;
 
-    public LimitViolation(Identifiable subject, LimitViolationType limitType, float limit, String limitName, float limitReduction, float value, Country country, float baseVoltage) {
+    public LimitViolation(@JsonProperty("subject") Identifiable subject, @JsonProperty("limitType") LimitViolationType limitType, 
```

Comment:
LimitViolation should be independent of the network. Subject should be replaced by a subjectId parameter.